### PR TITLE
arrays

### PR DESCRIPTION
**Pull request by PoisonDarterz**

### CODE REVIEW

**Review by LimJY03 — CHANGES_REQUESTED** [review]
## Consistency Fix Request

Please edit these to improve consistency:
1. One space after `//` 
    ```diff
    ! Line 34
    ! Line 35
    ! Line 37
    ! Line 94
    ! Line 117
    ! Line 118
    ! Line 153
    ```
2. Horizontal position of comments head
    ```diff
    ! Line 48
    ! Line 102
    ! Line 117
    ! Line 118
    ```
3. Comments for output should be at the same line of output code
    ```diff
    ! Line 153
    ```

File: 02. Inputs and Outputs/README.md [hunk 1]
```diff
@@ -0,0 +1,88 @@
+# Inputs and Outputs (Console)
+
+Lorem ipsum
+<-- rmb talk about escape characters, i believe most langs are almost the same for escape characters -->
```
Comment (LimJY03) [inline 02. Inputs and Outputs/README.md:4]: Just a side note, you forgot `!` after `<`, but I will edit that for you

**Review by LimJY03 — APPROVED** [review]
Thankss

File: 02. Inputs and Outputs/README.md (hunk 1 above)
Comment (PoisonDarterz) [reply 02. Inputs and Outputs/README.md:4]: oof lol ok

**Pull request merged**